### DISCUSSION TITLE
Hubs Cloud / offline

### DISCUSSION THREAD

**Discussion by degoao** [mozilla/hubs#4072] [Hubs Cloud Questions] [answered]
Keep on getting a combination of errors. Either:

* Hubs Cloud is currently offline. Check back shortly.

* Unable to connect to this room, please try again later.
You can try connecting via TCP, which may work better on some networks. (TCP doesn't work)

* And at somepoint I saw a 504 error.

No idea as to where to fix this as I am a notice to all these things. Hoping to have an opening for this exhibition tonight!!!

Any help would be very much appreciated.

[http://bwgalleries.xyz](url)

**Comment by johnshaughnessy** [comment, marked as answer]
@degoao is this still an issue you are seeing? I was able to connect to rooms on your server without any issues. 

Remove the "answer" tag from my response if this is still a problem that you'd like help solving.

**Reply by johnshaughnessy** [reply]
If this is still a problem, can you add more information along the lines of these other discussions? One known cause of seeing "Hubs cloud is currently offline" is running out of memory on the server.
https://github.com/mozilla/hubs/discussions/3792
https://github.com/mozilla/hubs/discussions/3892
